SocialFilter/sections/specific.txt

```diff
@@ -77,6 +77,7 @@ armidaleexpress.com.au,bendigoadvertiser.com.au,canberratimes.com.au,centralwest
 !
+portal.productboard.com###portal-card-detail div:has(> span > div > svg.pb-icon > path[d^="M7 4H4.6L8 0l3.4"])
```

Comment:
I think that would be enough:
`portal.productboard.com###portal-card-detail div:has(> span > div > svg.pb-icon)`